internal/model/model.go

```diff
@@ -796,11 +800,26 @@ func (m *Model) Request(deviceID protocol.DeviceID, folder, name string, offset
 	}
 	m.fmut.RLock()
 	fn := filepath.Join(m.folderCfgs[folder].Path(), name)
+	mtimeRepo, _ := m.folderMtimeRepos[folder]
 	m.fmut.RUnlock()
 
 	var reader io.ReaderAt
 	var err error
-	if info, err := os.Lstat(fn); err == nil && info.Mode()&os.ModeSymlink != 0 {
+	info, err := os.Lstat(fn)
```

Comment:
No, because it does not evaluate the 2nd part of the && if the first one is false.